README.md

```diff
@@ -48,6 +48,14 @@ Multiple SDKs are available for writing client applications that interface with
 * C++: [Docs](https://cpp.viam.dev/), [Repository](https://github.com/viamrobotics/viam-cpp-sdk/)
 * Rust: [Repository](https://github.com/viamrobotics/viam-rust-sdk)
 
+### Environment Variable Settings
+
+| **Environment Variable**            | **Behavior**                                               | **Default Value**    |
+|-------------------------------------|------------------------------------------------------------|----------------------|
+| VIAM_RESOURCE_CONFIGURATION_TIMEOUT | Duration for which resources are allowed to (re)configure. | 1 minute             |
+| VIAM_MODULE_STARTUP_TIMEOUT         | Duration for which modules are allowed to startup.         | 5 minutes            |
+| ENV                                 | If set to "development", runs the server in dev mode.      | Server runs normally |
```

Comment:
I don't really understand what `ENV=development` does from reading the code. Happy to remove this or alter the description if others know.